examples/text/text.cpp

```diff
@@ -75,6 +75,33 @@ void render(uint32_t time) {
   screen.pen = Pen(0xFF, 0xFF, 0xFF);
   screen.clip = clip;
   screen.text(text, minimal_font, text_rect, variable_width, TextAlign::center_center);
+  screen.clip = Rect(Point(0, 0), screen.bounds); // Reset the clip!
+
+  // Using a custom font
+  text_rect.x -= 160;
+  text_rect.y += 120;
+
+  screen.pen = Pen(64, 64, 64);
+  screen.rectangle(text_rect);
+
+  text = "This text uses\nan imported\nTrueType font.";
+  text = screen.wrap_text(text, text_rect.w, minimal_font, variable_width);
+
+  screen.pen = Pen(0xFF, 0xFF, 0xFF);
+  screen.text(text, press_start_font, text_rect, variable_width, alignment);
```

Comment:
Slightly surprised this works, but it's a single argument constructor so I guess it does... Probably not the most efficient to be constructing a `Font` every call though.

(I usually have the asset be `thing_font_data` and have a `const Font thing_font(thing_font_data)` somewhere.)

Reply:
Huh, you're right. I think it's a bit of a hangover from the early days when fonts were "just" int arrays. Will fix before @Gadgetoid wakes up, because examples should probably Do Things Properly...

Reply:
Might be an idea to mark that constructor `explicit` to avoid that...